<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor</class>
 <widget class="QMainWindow" name="TextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="Edit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_New"/>
   <addaction name="action_Save"/>
   <addaction name="action_SavaAs"/>
   <addaction name="separator"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Paste"/>
   <addaction name="action_Find"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Y"/>
   <addaction name="separator"/>
   <addaction name="action_Left"/>
   <addaction name="action_Center"/>
   <addaction name="action_Right"/>
   <addaction name="action_Just"/>
   <addaction name="separator"/>
   <addaction name="action_Underline"/>
   <addaction name="action_Bold"/>
   <addaction name="action_Italic"/>
   <addaction name="action_Font"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_SavaAs"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Y"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_selectAll"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menu_F_3">
     <property name="title">
      <string>字体(&amp;F)</string>
     </property>
     <addaction name="action_Bold"/>
     <addaction name="action_Italic"/>
     <addaction name="action_Underline"/>
     <addaction name="action_Font"/>
    </widget>
    <widget class="QMenu" name="menu_P">
     <property name="title">
      <string>段落(&amp;P)</string>
     </property>
     <addaction name="action_Left"/>
     <addaction name="action_Right"/>
     <addaction name="action_Center"/>
     <addaction name="action_Just"/>
    </widget>
    <addaction name="menu_F_3"/>
    <addaction name="menu_P"/>
    <addaction name="action_Color"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/new.png</normaloff>:/myImages/rsc/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/open.png</normaloff>:/myImages/rsc/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/close.png</normaloff>:/myImages/rsc/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/save.png</normaloff>:/myImages/rsc/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SavaAs">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/saveas.png</normaloff>:/myImages/rsc/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/close.png</normaloff>:/myImages/rsc/close.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/undo.png</normaloff>:/myImages/rsc/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/cut.png</normaloff>:/myImages/rsc/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/copy.png</normaloff>:/myImages/rsc/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/paste.png</normaloff>:/myImages/rsc/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/find.png</normaloff>:/myImages/rsc/find.png</iconset>
   </property>
   <property name="text">
    <string>查找/替换(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/help.png</normaloff>:/myImages/rsc/help.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_Bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/bold.png</normaloff>:/myImages/rsc/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗(&amp;B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_Italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/i.png</normaloff>:/myImages/rsc/i.png</iconset>
   </property>
   <property name="text">
    <string>倾斜(&amp;I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Underline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/underline.png</normaloff>:/myImages/rsc/underline.png</iconset>
   </property>
   <property name="text">
    <string>下划线(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_Left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/left.png</normaloff>:/myImages/rsc/left.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="action_Right">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/right.png</normaloff>:/myImages/rsc/right.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="action_Center">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/middle.png</normaloff>:/myImages/rsc/middle.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="action_Color">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/color.png</normaloff>:/myImages/rsc/color.png</iconset>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="action_Font">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/size.png</normaloff>:/myImages/rsc/size.png</iconset>
   </property>
   <property name="text">
    <string>设置字体</string>
   </property>
  </action>
  <action name="action_Just">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/just.png</normaloff>:/myImages/rsc/just.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
  </action>
  <action name="action_Y">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/redo.png</normaloff>:/myImages/rsc/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做(&amp;Y)</string>
   </property>
   <property name="toolTip">
    <string>重做(Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_selectAll">
   <property name="icon">
    <iconset resource="myresorces.qrc">
     <normaloff>:/myImages/rsc/selectall.png</normaloff>:/myImages/rsc/selectall.png</iconset>
   </property>
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>全选(A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Edit</class>
   <extends>QTextEdit</extends>
   <header>edit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myresorces.qrc"/>
 </resources>
 <connections/>
</ui>
